app/renderer/js/notification/helpers.ts

```diff
@@ -139,18 +145,8 @@ function parseReply(reply) {
 	return reply;
 }
 
-function setupReply(id) {
+export function setupReply(id: string): void {
 	const { narrow } = window;
 	const narrowByTopic = narrow.by_topic || narrow.by_subject;
 	narrowByTopic(id, { trigger: 'notification' });
 }
-
-module.exports = {
```

Comment:
Can't we follow the same thing we did earlier and do something like -
```
exports = { 1, 2 } 
```
here ?

Reply:
I prefer not to do that just for the sake of convention relating to this ES6 syntax.